### ISSUE TITLE
Unable to use TILT Temp for Controller Sensor

### ISSUE THREAD

**Issue by ChuckGl** [hrafnkelle/tfbrew#1]
I am hoping to use the temperature reported by TILT for controlling my TP-LINK providing power to the compressor.  I get an error regarding no attribs of 'temp' when trying to use the TILT Temperature for the sensor source for the FermentationFridge controller.

**Comment by hrafnkelle** [comment]
Can you provide your tfbrew.log file showing the error? 

What color is your Tilt?

**Comment by hrafnkelle** [comment]
I think I figured out what is causing this for you. Have changed the code, will commit it after I have tested it.

**Comment by hrafnkelle** [comment]
Hopefully fixed by commit b6fa574

Please pull the head from master to test.

**Comment by ChuckGl** [comment]
I'm afraid I am new to GitHub and am not sure what pull the head from the
master means.  Looks like it was tfbrew/plugins/TiltSensor/_init_.py that
changed so I am updating that on my box.

Do you still need my log files?

**Comment by hrafnkelle** [comment]
You can also just reinstall or manually apply the changes from TiltSensor/__init__.py

No logs not needed unless this doesn't work

**Comment by hrafnkelle** [comment]
but pull from head means just the command
git pull origin master
in the tfbrew/ directory (assuming you installed by doing git clone ...)

**Comment by ChuckGl** [comment]
Thanks the git pull origin master works much better than SFTP.  Everything
seems to be working now  Temperature is ready from the TILT and if I move
setpoint up or down from the TILT temperature the TP-Link powers on or off.


Could you tell me how I would control a second TP-Link for the plant heater
I have so I can turn on the Compressor when over temp and turn it off and
heater on when under?

Thanks  this is great little piece of code.

Chuck

**Comment by hrafnkelle** [comment]
Currently there is no support for that in tfbrew but it is on the TODO list. 
I'll see if I can get to it soon.

**Comment by ChuckGl** [comment]
Guess I just have to keep my temps at a steady until then.

Thanks again,
Chuck

**Comment by ChuckGl** [comment]
Just want to let you know. I got the heating working. I copied and modified the HysteresisLogic.py from >=setpoint to <= setpoint. I then created a Controller using the modified Hysteresis script and dummy actor for actor to test. I set cooling setpoint to 1 degree over the setpoint for heating. Each will be tied to a tplink to turn on/off the fridge and heating pad as needed. 

I am sure there is a much more elegant way to do it maybe even combining cooling and heating in one controller but this works for me and I’m more hack than programmer. 

Chuck
